justfile

```diff
@@ -0,0 +1,46 @@
+set dotenv-load := false
+
+DOCKER_FILE := "-f docker-compose.yml"
```

Comment:
Good to have in case we split to multiple Compose files. Thanks for future-proofing this!